optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -336,349 +213,29 @@ class ORTEncoderForWhisper(ORTEncoder):
             The ONNX Runtime inference session associated to the encoder.
     """
 
-    def prepare_io_binding(
-        self,
-        input_features: torch.FloatTensor = None,
-    ):
-        io_binding = self.session.io_binding()
-
-        # bind input features
-        input_features = input_features.contiguous()
-        io_binding.bind_input(
-            "input_features",
-            input_features.device.type,
-            self._device.index,
-            self.name_to_np_type["input_features"],
-            tuple(input_features.shape),
-            input_features.data_ptr(),
-        )
-
-        # bind logits
-        output_shape, output_buffer = self.prepare_output_buffer(
-            batch_size=input_features.size(0),
-            sequence_length=input_features.size(2) // 2,
```

Comment:
How are you handling this case?

Reply:
It is not handled. But as soon as this is merged, will work on a solution for that. The idea is to keep this PR as small as possible.